Sources/NIOHTTPCompression/HTTPResponseDecompressor.swift

```diff
@@ -0,0 +1,207 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
```

Comment:
copyright doesn't contain 2019. If this is a new thing, just put '2019', if this contains old code it should be 2017-2019

Reply:
fixed!